tests/x11/smt.pm

```diff
@@ -7,18 +7,19 @@
 # notice and this notice are preserved.  This file is offered as-is,
 # without any warranty.
 
-# G-Summary: Add smt configuration test
+# Summary: Add smt configuration test
 #    test installation and upgrade with smt pattern, basic configuration via
 #    smt-wizard and validation with smt-repos smt-sync return value
-# G-Maintainer: Jozef Pupava <jpupava@suse.com>
+# Maintainer: Jozef Pupava <jpupava@suse.com>
 
 use base "x11test";
 use strict;
 use warnings;
 use testapi;
 
 sub run() {
-    x11_start_program("xterm -geometry 150x35+5+5");
+    x11_start_program("xterm");
```

Comment:
[this](http://dhcp91.suse.cz/tests/2866#step/smt/21) is better readable than in a bigger xterm ?